docs/source/usage/common_scenarios.rst

```diff
@@ -0,0 +1,72 @@
+Common Scenarios
+================
+
+Lessee Node Assignment
+----------------------
+
+Lessee node assignment can be performed by the administrator or node owner. It can be executed using the Ironic CLI:
+
+  .. prompt:: bash $
+
+    openstack baremetal node set –lessee <lessee project id> <node>
```

Comment:
Isn't it supposed to be --lessee?

Reply:
Whoops - I think copying and pasting from the google doc messed things up. Fixed!